dashboardv2/api.go

```diff
@@ -0,0 +1,284 @@
+package main
+
+import (
+	"bytes"
+	"crypto/subtle"
+	"encoding/json"
+	"fmt"
+	"mime"
+	"net/http"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/flynn/flynn/pkg/cors"
+	"github.com/flynn/flynn/pkg/ctxhelper"
+	"github.com/flynn/flynn/pkg/httphelper"
+	"github.com/flynn/flynn/pkg/status"
+	gctx "github.com/gorilla/context"
+	"github.com/gorilla/sessions"
+	"github.com/julienschmidt/httprouter"
+	"golang.org/x/net/context"
+)
+
+type LoginInfo struct {
+	Token string `json:"token"`
+}
+
+func NewDashboardHandler(conf *Config) http.Handler {
+	api := &API{
+		conf:       conf,
+		assetCache: make(map[string][]byte),
+	}
+
+	router := httprouter.New()
+	router2 := httprouter.New()
+
+	prefixPath := func(p string) string {
+		return p // TODO(jvatic): handle path prefix?
```

Comment:
Can this be dropped?

Reply:
Yeah, I don't think need the ability to mount the dashboard any deeper than to domain root.